pkg/helm/io_test.go

```diff
@@ -0,0 +1,215 @@
+package helm
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func Test_Load(t *testing.T) {
+	tests := []struct {
+		name    string
+		content string
+		want    ValuesMap
+		err     bool
+	}{
+		{
+			name: "valid yaml file",
+			content: `
```

Comment:
There should also be a test for when `image:` is at the top level of the file for things like inlets-operator.

Reply:
updated content with a top level `image` key.